.github/workflows/ci.yml

```diff
@@ -108,14 +106,10 @@ jobs:
       - name: Self install
         run: |
           cargo install svd2rust --path .
+          cargo install svd2rust-regress --path ci/svd2rust-regress
```

Comment:
this is uneccessary, can just call the alias `cargo regress`

Reply:
I thought it might be fast if it is installed first before running it, but I don´t think it is..